validate/validators/bam.py

```diff
@@ -1,5 +1,6 @@
 '''Helper methods for bam file validation'''
 
+from xml.dom import ValidationErr
```

Comment:
It doesn't look like this is used.

Reply:
I'm not sure where that came from! I'll take it out.